packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -1033,30 +1024,15 @@ export const Table = <
                     });
```

Comment:
I think all this code above can also be moved into `ColumnHeader`

Reply:
Moved this code into `ColumnHeader` with this [commit](https://github.com/iTwin/iTwinUI/pull/2178/commits/0198db023ca2cec3cad5eed4b1162ee5b3275cc0).